torch/_functorch/aot_autograd.py

```diff
@@ -1659,8 +1659,29 @@ def _collect_assigned_tensor_attributes(kp, v, _v):
                 # TODO(avik): Assigning all other types are allowed right now.
                 # Maybe in the future we want to limit this to primitive types?
 
+        new_attrs = _get_attributes(mod)
+        if len(new_attrs) != len(snapshot):
+            added_attrs = new_attrs.keys() - snapshot.keys()
+            deleted_attrs = snapshot.keys() - new_attrs.keys()
+
+            if len(added_attrs) > 0:
+                raise ValueError(
+                    f"During torch.export, following attrs were created in the model.forward: {added_attrs} "
```

Comment:
`following attrs were created in the model.forward` -> `the following attributes were created`?